### PR TITLE
reorg manure module

### PR DESCRIPTION
**Pull request by PooyaHekmati**
Doing a few things in the Manaure Module with no logic change (this is not everything that needs to be modified):

1. Renamed GasEmissions class to GasEmissionsCalculator
2. Updated a few f-strings
3. Applied auto-formatting -- this is why the PR looks massive but it is not
4. Removed a few unused info_maps and imports
5. Restructured the directory and files as follows:

Before:

> .
> │   manure_manager.py
> │   __init__.py
> │
> ├───beddings
> │   │   bedding_classes.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           bedding_classes.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───constants
> │       gas_emission_constants.py
> │       manure_constants.py
> │       __init__.py
> │
> ├───constants_and_units
> │   └───__pycache__
> │           gas_emission_constants.cpython-311.pyc
> │           manure_constants.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───default_enum
> │   │   default_enum.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           default_enum.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───gas_emissions
> │   │   gas_emissions.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           calculator.cpython-311.pyc
> │           gas_emissions.cpython-311.pyc
> │           gas_emissions_calculator.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───input_handler
> │       manure_manager_config_handler.py
> │       __init__.py
> │
> ├───IO_helpers
> │   └───__pycache__
> │           manure_manager_config_handler.cpython-311.pyc
> │           manure_module_output_manager_helper.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───manure_handlers
> │   │   manure_handler_classes.py
> │   │   manure_handler_daily_output.py
> │   │   milking_parlor.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           manure_handler_classes.cpython-311.pyc
> │           manure_handler_daily_output.cpython-311.pyc
> │           milking_parlor.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───manure_nutrients
> │   │   manure_nutrients.py
> │   │   manure_nutrient_manager.py
> │   │   nutrient_request.py
> │   │   nutrient_request_results.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           manure_nutrients.cpython-311.pyc
> │           manure_nutrient_manager.cpython-311.pyc
> │           nutrient_request.cpython-311.pyc
> │           nutrient_request_results.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───manure_separators
> │   │   manure_separator_classes.py
> │   │   manure_separator_daily_output.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           manure_separator_classes.cpython-311.pyc
> │           manure_separator_daily_output.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───manure_treatments
> │   │   anaerobic_digestion.py
> │   │   anaerobic_digestion_and_lagoon.py
> │   │   anaerobic_lagoon.py
> │   │   base_manure_treatment.py
> │   │   manure_treatment_cbpb.py
> │   │   manure_treatment_configs.py
> │   │   manure_treatment_daily_output.py
> │   │   manure_treatment_factory.py
> │   │   manure_treatment_types.py
> │   │   manure_types.py
> │   │   slurry_storage_outdoor.py
> │   │   slurry_storage_underfloor.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           anaerobic_digestion.cpython-311.pyc
> │           anaerobic_digestion_and_lagoon.cpython-311.pyc
> │           anaerobic_lagoon.cpython-311.pyc
> │           base_manure_treatment.cpython-311.pyc
> │           manure_treatment_cbpb.cpython-311.pyc
> │           manure_treatment_configs.cpython-311.pyc
> │           manure_treatment_daily_output.cpython-311.pyc
> │           manure_treatment_factory.cpython-311.pyc
> │           manure_treatment_types.cpython-311.pyc
> │           manure_types.cpython-311.pyc
> │           slurry_storage_outdoor.cpython-311.pyc
> │           slurry_storage_underfloor.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───output_manager_helper
> │       manure_module_output_manager_helper.py
> │       __init__.py
> │
> ├───pen
> │   │   manure_manager_pen.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           manure_manager_pen.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───pen_manure
> │   │   pen_manure.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           manure_manager_pen.cpython-311.pyc
> │           pen_manure.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───protocols
> │   │   liquid_manure_portion_protocol.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           liquid_manure_portion_protocol.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───reception_pits
> │   │   reception_pit.py
> │   │   reception_pit_daily_output.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           reception_pit.cpython-311.pyc
> │           reception_pit_daily_output.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───units
> │       units.py
> │       __init__.py
> │
> └───__pycache__
>         manure_manager.cpython-311.pyc
>         __init__.cpython-311.pyc

		
	
		
After		

> .
> │   manure_manager.py
> │   __init__.py
> │
> ├───beddings
> │   │   bedding_classes.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           bedding_classes.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───constants_and_units
> │   │   gas_emission_constants.py
> │   │   manure_constants.py
> │   │   units.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           gas_emission_constants.cpython-311.pyc
> │           manure_constants.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───default_enum
> │   │   default_enum.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           default_enum.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───gas_emissions
> │   │   calculator.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           calculator.cpython-311.pyc
> │           gas_emissions.cpython-311.pyc
> │           gas_emissions_calculator.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───IO_helpers
> │   │   manure_manager_config_handler.py
> │   │   manure_module_output_manager_helper.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           manure_manager_config_handler.cpython-311.pyc
> │           manure_module_output_manager_helper.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───manure_handlers
> │   │   manure_handler_classes.py
> │   │   manure_handler_daily_output.py
> │   │   milking_parlor.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           manure_handler_classes.cpython-311.pyc
> │           manure_handler_daily_output.cpython-311.pyc
> │           milking_parlor.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───manure_nutrients
> │   │   manure_nutrients.py
> │   │   manure_nutrient_manager.py
> │   │   nutrient_request.py
> │   │   nutrient_request_results.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           manure_nutrients.cpython-311.pyc
> │           manure_nutrient_manager.cpython-311.pyc
> │           nutrient_request.cpython-311.pyc
> │           nutrient_request_results.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───manure_separators
> │   │   manure_separator_classes.py
> │   │   manure_separator_daily_output.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           manure_separator_classes.cpython-311.pyc
> │           manure_separator_daily_output.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───manure_treatments
> │   │   anaerobic_digestion.py
> │   │   anaerobic_digestion_and_lagoon.py
> │   │   anaerobic_lagoon.py
> │   │   base_manure_treatment.py
> │   │   manure_treatment_cbpb.py
> │   │   manure_treatment_configs.py
> │   │   manure_treatment_daily_output.py
> │   │   manure_treatment_factory.py
> │   │   manure_treatment_types.py
> │   │   manure_types.py
> │   │   slurry_storage_outdoor.py
> │   │   slurry_storage_underfloor.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           anaerobic_digestion.cpython-311.pyc
> │           anaerobic_digestion_and_lagoon.cpython-311.pyc
> │           anaerobic_lagoon.cpython-311.pyc
> │           base_manure_treatment.cpython-311.pyc
> │           manure_treatment_cbpb.cpython-311.pyc
> │           manure_treatment_configs.cpython-311.pyc
> │           manure_treatment_daily_output.cpython-311.pyc
> │           manure_treatment_factory.cpython-311.pyc
> │           manure_treatment_types.cpython-311.pyc
> │           manure_types.cpython-311.pyc
> │           slurry_storage_outdoor.cpython-311.pyc
> │           slurry_storage_underfloor.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───pen
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           manure_manager_pen.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───pen_manure
> │   │   manure_manager_pen.py
> │   │   pen_manure.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           manure_manager_pen.cpython-311.pyc
> │           pen_manure.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───protocols
> │   │   liquid_manure_portion_protocol.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           liquid_manure_portion_protocol.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> ├───reception_pits
> │   │   reception_pit.py
> │   │   reception_pit_daily_output.py
> │   │   __init__.py
> │   │
> │   └───__pycache__
> │           reception_pit.cpython-311.pyc
> │           reception_pit_daily_output.cpython-311.pyc
> │           __init__.cpython-311.pyc
> │
> └───__pycache__
>         manure_manager.cpython-311.pyc
>         __init__.cpython-311.pyc

### CODE REVIEW

File: RUFAS/routines/manure/IO_helpers/manure_manager_config_handler.py
Comment (allisterakun) [inline RUFAS/routines/manure/IO_helpers/manure_manager_config_handler.py:1]: All changes are formatting in this file.

File: RUFAS/routines/manure/IO_helpers/manure_module_output_manager_helper.py
Comment (allisterakun) [inline RUFAS/routines/manure/IO_helpers/manure_module_output_manager_helper.py:1]: All changes are formatting in this file.

File: RUFAS/routines/manure/beddings/bedding_classes.py
Comment (allisterakun) [inline RUFAS/routines/manure/beddings/bedding_classes.py:1]: All changes are formatting in this file.

File: RUFAS/routines/manure/default_enum/default_enum.py
Comment (allisterakun) [inline RUFAS/routines/manure/default_enum/default_enum.py:1]: All changes are formatting in this file.

File: RUFAS/routines/manure/manure_handlers/manure_handler_classes.py
Comment (allisterakun) [inline RUFAS/routines/manure/manure_handlers/manure_handler_classes.py:1]: Auto-formatting and GasEmissionsCalculator renaming

File: RUFAS/routines/manure/gas_emissions/calculator.py
Comment (allisterakun) [inline RUFAS/routines/manure/gas_emissions/calculator.py:1]: Formatting and renaming classes/methods

File: tests/test_manure_module/test_pen_manure.py
Comment (allisterakun) [inline tests/test_manure_module/test_pen_manure.py:1]: Import file path change.

File: tests/test_manure_module/test_milking_parlor.py
Comment (allisterakun) [inline tests/test_manure_module/test_milking_parlor.py:1]: Import file path change.

File: tests/test_manure_module/test_manure_separators.py
Comment (allisterakun) [inline tests/test_manure_module/test_manure_separators.py:1]: Import file path change.

File: RUFAS/routines/manure/manure_handlers/manure_handler_daily_output.py
Comment (allisterakun) [inline RUFAS/routines/manure/manure_handlers/manure_handler_daily_output.py:1]: Auto-formatting

File: RUFAS/routines/manure/manure_handlers/milking_parlor.py
Comment (allisterakun) [inline RUFAS/routines/manure/manure_handlers/milking_parlor.py:1]: Import file path change and auto-formatting.

File: RUFAS/routines/manure/manure_manager.py
Comment (allisterakun) [inline RUFAS/routines/manure/manure_manager.py:1]: Formatting

File: tests/test_manure_module/test_reception_pits.py
Comment (allisterakun) [inline tests/test_manure_module/test_reception_pits.py:1]: Formatting

File: tests/test_manure_module/test_manure_treatments.py
Comment (allisterakun) [inline tests/test_manure_module/test_manure_treatments.py:1]: Formatting.

File: tests/test_manure_module/test_manure_manager_constants.py
Comment (allisterakun) [inline tests/test_manure_module/test_manure_manager_constants.py:1]: Import path.

**Review by allisterakun — APPROVED** [review]
Lots of small and simple changes.

**Review by KFosterReed — APPROVED** [review]
I just checked if the manure output values are the same and they are so looks good to me!

**Review by JoeWaddell — APPROVED** [review]
I think the structural changes make sense. Changes to the code looks good, and I didn't find any discrepancies in the outputs.

**Review by ltp23 — APPROVED** [review]
LGTM. Thanks, Pooya.

**Pull request merged**